conversions/bmi_conversion.py

```diff
@@ -0,0 +1,37 @@
+"""Converts height and mass to Body Mass Index"""
+
+
+def metric_units_bmi(kg: int, cm: int) -> int:
+    """
+    Convert mass in kilograms and height in centimetres to Body Mass Index
+    >>> metric_units_bmi(50, 160)
+    19.53
+    >>> metric_units_bmi(80, 175)
+    26.12
+    """
+    metre_square = (cm / 100) ** 2
+    return round(kg / metre_square, 2)
+
+
+if __name__ == "__main__":
+    import doctest
+
+    doctest.testmod()
+
+
+def us_units_bmi(pounds: int, inches: int) -> int:
```

Comment:
It will raise an exception if `inches==0`

Reply:
Why resolved? You didn't fix it

Reply:
@mateuszz0000 
Please check again. 
From my side I have fixed it. I have also viewed the file inside this pull request. There is now only one "if main" on line 28. 

If this is not the case please tell me where I am wrong.